Documentation/api/ClientHowto.md

```diff
@@ -65,6 +65,16 @@ Delve will respond by sending a response packet that will look like this:
 {"id":27, "result": {"Breakpoint": {"id":3, "name":"", "addr":4538829, "file":"/User/you/some/file.go", "line":16, "functionName":"main.main", "Cond":"", "continue":false, "goroutine":false, "stacktrace":0, "LoadArgs":null, "LoadLocals":null, "hitCount":{}, "totalHitCount":0}}, "error":null}
 ```
 
+## Selecting the API version
+
+Delve currently supports two version of its API, APIv1 and APIv2. By default
+a headless instance of `dlv` will serve APIv1, however new clients should
```

Comment:
It is a bit odd that the default is not the recommended version. I think it is worth being explicit about it in the documentation by adding something like "will serve APIv1 for backward-compatibility with older clients".

Reply:
Done.